src/tensorflow_object_detector/data/labels/mscoco_label_map.pbtxt

```diff
@@ -0,0 +1,400 @@
+item {
```

Comment:
This `data` folder doesn't seem like something we need to own? Unless we plan on modifying it? Is this part of the tensorflow models repo?